packages/lesmis/src/views/DashboardView.js

```diff
@@ -39,108 +61,341 @@ const RedTitle = styled(Typography)`
   padding-top: 50px;
 `;
 
+const GreyTitle = styled(Typography)`
+  color: ${props => props.theme.palette.text.secondary};
+  font-weight: bold;
+  padding-top: 30px;
+  padding-left: 5px;
+`;
+
+const HorizontalDivider = styled(MuiDivider)`
+  width: 90%;
+  margin-top: 1.5rem;
+  background: ${props => props.theme.palette.text.tertiary};
+`;
+
+const VerticalDivider = styled(MuiDivider)`
+  margin-top: 2.5rem;
+  height: 7rem;
+  background: ${props => props.theme.palette.text.tertiary};
+`;
+
+const PlaceholderBox = styled.div`
+  width: 510px;
+  height: 370px;
+  background: grey;
+`;
+const MapPlaceholder = () => (
+  <PlaceholderBox>
+    <Typography>TODO: Map</Typography>
+  </PlaceholderBox>
+);
+
+const CountryView = ({ vitals }) => {
+  return (
+    <Container maxWidth={false}>
```

Comment:
```suggestion
const CountryView = ({ vitals }) =>  (
    <>
```

Reply:
Might as well pull the container out to the main component to save a few lines of code

Reply:
Ah cool, I tried splitting it out but it kept complaining that I needed to return a single component. Using an empty component was the concept I was missing.